src/wix/WixToolset.Core/Linker.cs

```diff
@@ -155,6 +148,17 @@ public Intermediate Link(ILinkContext context)
                     return null;
                 }
 
+                // If there are no authored features, assign the referenced components to
+                // the StdLib default feature, then re-run resolve to pick up the default
+                // feature from the StdLib.
+                {
+                    var command = new AssignDefaultFeatureCommand(find, resolve.ResolvedSections);
+                    command.Execute();
+                }
+
+                resolve = new ResolveReferencesCommand(this.Messaging, find.EntrySection, find.SymbolsByName);
+                resolve.Execute();
```

Comment:
The other commits look fine but I want to look at a different solution than calling resolve references twice. This could introduce lots of complexity.

Reply:
Please enjoy. Would you like an issue to track?

Reply:
We should always open an issue to track since we'll forget about the issue otherwise. 😄